chaoslib/openebs/fetch_data_from_replica.yml

```diff
@@ -0,0 +1,63 @@
+- name: Get replica id
+  shell: kubectl get pods {{ rpod }} -n openebs -o jsonpath='{.spec.containers[*].env[0].value}'
+  register: cstor_id
+
+- name: Check for dataset name
+  shell: kubectl exec -it {{ rpod }} -n  openebs --container cstor-pool -- zfs list cstor-{{ cstor_id.stdout }}/{{ pv.stdout }}
+  register: ds_exists
```

Comment:
do you mean to do a check on the volname from ds_exists.stdout ?

Reply:
Yes.
Earlier, I was taking a snapshot through `zfs` commands. Due to that, i have added this check for checking if a dataset exists.